### ISSUE TITLE
`ks_spectrum` test comparisons use different input file formats

### ISSUE THREAD

**Issue by smangham** [milc-qcd/milc_qcd#66]
I'm attempting to build and test `milc`, but can't really tell if it's working as the tests appear to be based on a much older version of the code.

Building `ks_spectrum` and running `make check "EXEC=ks_spectrum_hisq" &> make_test.log`, all the tests fail. As an example, `ks_spectrum_hisq.nd1.test-out` ends up with:
```
get_i: ERROR IN INPUT: expected max_cg_iterations but found set_type
exit: Thu Jul 18 16:27:06 2024
```
Comparing the generated output and the sample output, it looks like there's substantial differences:
![Screenshot 2024-07-18 164728](https://github.com/user-attachments/assets/c949f8c8-9403-4f2b-8b1f-c0fe0c920499)
![Screenshot 2024-07-18 164752](https://github.com/user-attachments/assets/c487fda9-6d58-433d-9cf8-dcc05c4b3029)

It looks like these arise at least in part because the input file `ks_spectrum_hisq.nd1.test-out` does not actually reflect the file that's *supposed* to be built by `build_input.ks_spectrum_hisq.nd.1.sh` - the `sample.in` file has a `set_type multimass` line that `build_input` no longer uses. I'm not sure which command or part of the make chain would actually force a rebuild of those `sample.in` files.

I'll try just deleting them to force a rebuild, but I think they should probably be either removed from the repo, or the makefiles should be adjusted to force an update.

**Comment by smangham** [comment]
Right, I think I had it back to front - deleting the `set_type multimass` line from the `sample-in` files causes the tests to pass.

**Comment by detar** [comment]
I hope you are using the "develop" branch.
